Engine/lib/convexDecomp/NvThreadConfig.cpp

```diff
@@ -129,26 +133,22 @@ void   tc_sleep(NxU32 ms)
 
 void tc_spinloop()
 {
-#if defined( _XBOX )
-   // Pause would do nothing on the Xbox. Threads are not scheduled.
-#elif defined( _WIN64 )
-   YieldProcessor( );
-#elif defined( __APPLE__ )
-   pthread_yield_np();
-#elif defined(__linux__)
-   #if defined(_POSIX_PRIORITY_SCHEDULING)
-      sched_yield();
+   #if defined(__linux__) || defined( __APPLE__ ) ||  defined( __FreeBSD__)
+
+      asm ( "pause" );
```

Comment:
Pretty sure we've already previously had discussions about using inlined x86 assembly instruction, vs using the preferred way of involving the OS scheduler in yielding the task and for cross architecture compatibility (`pause` doesn't exist for ARM, etc..).